js/viewer.js

```diff
@@ -8,7 +8,8 @@ if (typeof String.prototype.endsWith !== "function")
         return this.indexOf(suffix, this.length - suffix.length) !== -1;
     };
 
-var local = location.href.indexOf('://www.cyberbotics.com/doc') == -1;
+var local = location.href.indexOf('://www.cyberbotics.com/doc') == -1 &&
+            location.href.indexOf('://robotbenchmark.net/webots_documentation') == -1 ;
```

Comment:
It is also a pity that cyberbotics.com is hard coded.
It depends on what is the purpose of this file.
To me it seems much better if the local variable is set in the setup variable. But this would require to change cyberbotics.com